react/utils/CurrentPosition.js

```diff
@@ -1,37 +1,38 @@
 import { geolocationAutoCompleteAddress } from 'vtex.address-form/geolocationAutoCompleteAddress'
 
 export function getCurrentPosition(successCallback, errorCallback) {
-  return navigator.geolocation.getCurrentPosition(
-    position => successCallback(position),
-    error => {
-      // TODO#1: retry in case of timeout, possibly with a lower accuracy.
```

Comment:
Looked into this, it's not worth retrying this because it has the same effect as just increasing the timeout